docs/4.tutorials-and-examples/6.custom-webchat.mdx

```diff
@@ -13,51 +13,86 @@ This template shows how to create custom elements for webchat. All the custom co
 **src/webchat/index.js**
 
 ```javascript
-import Logo from '../assets/bot_vader.jpg'
-import { MyReply } from './myReply'
-
-export const theme = {
-  brandColor: '#ff0000',
-  textPlaceholder: 'Please, type your message:',
-  title: 'Custom Webchat',
-  brandIconUrl: Logo,
-  customReply: MyReply
+export const webchat = {
+  theme: {},
+
+  persistentMenu: [],
+
+  blockInputs: [],
+
+  // Webchat listeners
+  onInit: app => {},
+  onOpen: app => {},
+  onClose: app => {},
+  onMessage: app => {}
 }
 ```
 
-This is the main file of the webchat config. In this file, we export all the options that are going to be changed of the default webchat config.
+This is just a boilerplate code containing all the allowed definitions for your Webchat.
```

Comment:
This is just a boilerplate code containing all the allowed definitions for your Webchat.=>
See above a template with Webchat allowed definitions. With them you can configure the following (see detailed examples below):

Reply:
updated